.ci/dockerfiles/Dockerfile.sglang-base

```
@@ -0,0 +1,55 @@
+ENV NIXL_SANITY_MODEL_ID=${MODEL_ID}
+ENV NIXL_SANITY_MODEL_DIR=${MODEL_DIR}
+ENV HF_HOME=${MODEL_DIR}/hf-home
+
+# iproute2 provides `ss`, used by the sanity's get_next_tcp_port helper to skip occupied
+# host ports (concurrent jobs share a SLURM node under host networking).
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends iproute2 \
+ && rm -rf /var/lib/apt/lists/*
+
+# Use the `hf` CLI (huggingface-cli is removed in recent huggingface_hub).
+RUN python3 -m pip install --no-cache-dir --upgrade "huggingface_hub[cli]" \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)